src/CheckoutSdk/Payments/Hosted/IHostedPaymentsClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.Payments.Hosted
+{
+	public interface IHostedPaymentsClient
+	{
+		Task<HostedPaymentResponse> CreateAsync(HostedPaymentRequest hostedPaymentRequest, CancellationToken cancellationToken = default);
```

Comment:
Please rename to `Create`. We have decided a while ago to drop `Async` in naming because all the methods are async.